### PR TITLE
Add command registrations to token and interoperability modules

### LINKED ISSUE #7819: Missing cross chain command property on token and interoperability module
### Expected behavior
Token module should register cross chain transfer command

### Actual behavior
It's implemented but not defined

### Which version(s) does this affect? (Environment, OS, etc...)
alpha.6-

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #7819 

### How was it solved?

- Add command registrations

### How was it tested?

- Check all the commands in the cc_commands and commands are registered to the module

### CODE REVIEW

File: framework/src/modules/interoperability/mainchain/module.ts [hunk 1]
```diff
@@ -79,11 +86,58 @@ export class MainchainInteroperabilityModule extends BaseInteroperabilityModule
...
-		this._sidechainRegistrationCommand,
+		this._crossChainUpdateCommand,
 		this._messageRecoveryInitializationCommand,
+		this._messageRecoveryCommand,
+		this._sidechainRegistrationCommand,
+		this._stateRecoveryCommand,
+		this._terminateSidechainForLivenessCommand,
+	];
+	// eslint-disable-next-line @typescript-eslint/member-ordering
+	public crossChainCommand = [
```
Comment (Phanco) [inline framework/src/modules/interoperability/mainchain/module.ts:128]: Would it be better if called `crossChainCommands` if it's an array?

File: framework/src/modules/token/module.ts [hunk 2]
```diff
@@ -76,6 +76,7 @@ export class TokenModule extends BaseInteroperableModule {
 	public endpoint = new TokenEndpoint(this.stores, this.offchainStores);
 	public crossChainMethod = new TokenInteroperableMethod(this.stores, this.events);
 	public crossChainTransferCommand = new CrossChainTransferMessageCommand(this.stores, this.events);
+	public crossChainCommand = [this.crossChainTransferCommand];
```
Comment (Phanco) [inline framework/src/modules/token/module.ts:79]: same as above

**Review by Phanco — APPROVED** [review]

**Comment by Phanco** [comment]
LGTM, but a small suggestion

File: framework/src/modules/interoperability/mainchain/module.ts (hunk 1 above)
Comment (shuse2) [reply framework/src/modules/interoperability/mainchain/module.ts:128]: yes, ill create another issue. This was defined in base class

File: framework/src/modules/interoperability/base_state_recovery.ts [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+import { computeStorePrefix } from '../base_store';
+import { BaseCCMethod } from './base_cc_method';
+import { BaseInteroperabilityInternalMethod } from './base_interoperability_internal_methods';
+
+export class BaseStateRecoveryCommand<
+	T extends BaseInteroperabilityInternalMethod
+> extends BaseInteroperabilityCommand<T> {
+	public schema = stateRecoveryParamsSchema;
+
+	public async verify(
```
[the remark this answers is not in the document]
Comment (shuse2) [reply framework/src/modules/interoperability/base_state_recovery.ts:38]: that is not the specification, and in general the scope of this PR is just definition, so we can improve comments or refactor in other PRs

**Pull request merged**